doc/source/tools/postprocessing/postprocessing_paraview.rst

```diff
@@ -0,0 +1,51 @@
+========================================
+Massive Data Visualization with Paraview
+========================================
+
+Lethe is developed with special care for scalability; it is intended to be used for solving extremely large problems by using massively parallel computing. As such, the output files that are obtained can sometimes be so heavy that the post-processing requires special considerations. This section covers key points to take into account in such cases.
+
+
+------------------------------
+Client-server visualization
+------------------------------
+
+The first limitation that is encountered when FEM problems contain :math:`\sim 15` M cells (when using a 32GB-RAM machine) is lack of memory. Even though the VTU files weigh 2GB, they are decompressed by Paraview when opened.
+
+
+Clusters such as those of the Digital Research Alliance of Canada allow to launch interactive jobs in which a Paraview-server can be used. Detailed instructions on how to setup and use such a server are available on this `page <https://docs.alliancecan.ca/wiki/ParaView>`_. However, here is an overview of the required steps to use Client-server visualization:
+
+
+
+1. Open 3 terminals: #1 will be used to run the server, #2 will be used to link the local-remote ports and #3 will be used to run the client.
+
+2. Ensure that the files to visualize are on the cluster. ``scp`` can be used to move them there.

```

Comment:
Is this necessary? If we are using the server, I guess the files should be there.

Reply:
I put "Log in to the cluster where the data is located." in the step 2.1. I think that covers it.